app/assets/stylesheets/welcome.scss

```diff
@@ -2,3 +2,40 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 $backdrop : #C9D6AE;
+
+.welcome {
+  background-image: image-url('skyscrapers.jpg');
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
```

Comment:
Properties should be ordered background-image, background-repeat, -moz-background-size, -o-background-size, -webkit-background-size, background-size, min-height, min-width<br>Avoid vendor prefixes.